go2_simulation/simple_wrapper.py

```diff
@@ -174,16 +174,12 @@ def addSystemCollisionPairs(model, geom_model, qref):
 class SimpleWrapper():
     def __init__(self, node, timestep):
         ########################## Load robot model and geometry
-        robot = erd.load("go2")
+        robot = loadGo2()
         self.rmodel = robot.model
         self.q0 = self.rmodel.referenceConfigurations["standing"]
         self.njoints = self.rmodel.nv - 6
 
-        URDF_SUBPATH = "/go2_description/urdf/go2.urdf"
-        package_dir = erd.getModelPath(URDF_SUBPATH)
-        file_path = package_dir + URDF_SUBPATH
-
-        with open(file_path, 'r') as file:
+        with open(GO2_DESCRIPTION_URDF_PATH, 'r') as file:
```

Comment:
Need to add `from go2_description import GO2_DESCRIPTION_URDF_PATH` to simple_wrapper.py

Reply:
Ah yes, since I didn't have access to simple I couldn't test.. Thank you